pkg/acceptance/bettertestspoc/config/model/view_model_gen.go

```diff
@@ -12,6 +12,7 @@ import (
 type ViewModel struct {
 	AggregationPolicy  tfconfig.Variable `json:"aggregation_policy,omitempty"`
 	ChangeTracking     tfconfig.Variable `json:"change_tracking,omitempty"`
+	Column             tfconfig.Variable `json:"column,omitempty"`
```

Comment:
Please generate the model despite the fact that the list is required and even use it by utilizing the `ConfigVariablesFromModel`

Reply:
Done.